validation/scripts/validate-plot-protein-lengths.R

```diff
@@ -46,40 +40,34 @@ diamond_proportion_plot <- diamond_tables_clean %>%
 
 diamond_proportion_plot
 
+# number of proteins with greater than 90% length proportion of reference hit
 diamond_tables_clean %>%
   left_join(tick_species_metadata) %>% 
   mutate(proportion = slen / qlen) %>% 
-  filter(proportion >=0.9) %>% 
+  filter(proportion > 0.9) %>% 
   group_by(species_name) %>% 
   count() %>% 
   left_join(tick_species_metadata) %>% 
   mutate(proportion_of_reference = n / total_protein_count) %>% 
   arrange(desc(proportion_of_reference))
 
-diamond_tables_clean %>%
-  left_join(tick_species_metadata) %>% 
-  mutate(proportion = slen / qlen) %>% 
-  filter(proportion <0.9) %>% 
-  group_by(species_name) %>% 
+# amblyomma proteins with more than 2 hits
+diamond_tables_clean %>% 
+  group_by(species_name, sseqid) %>%
   count() %>% 
-  left_join(tick_species_metadata) %>% 
-  mutate(proportion_of_reference = n / total_protein_count) %>% 
-  arrange(desc(proportion_of_reference))
-
-# combine with tick metadata for total protein counts
-tick_species_protein_hit_counts <- diamond_tables_clean %>% 
+  filter(n > 2) %>% 
   group_by(species_name) %>% 
   count() %>% 
-  mutate(protein_hits = n) %>% 
-  select(-n)
+  mutate(proportion_of_amblyomma = (n / 28319) * 100) # divided by total proteins in A. americanum for more than 2 hits
+
 
 tick_species_metadata_hits <- left_join(tick_species_protein_hit_counts, tick_species_metadata) %>% 
   mutate(proportion_of_reference = protein_hits / total_protein_count) %>% 
-  mutate(proportion_of_amblyomma_query = protein_hits / 34557) %>% 
+  mutate(proportion_of_amblyomma_query = protein_hits / 28319) %>% 
```

Comment:
do you want to code these as numbers instead of hard coding so they auto update? no worries if not since this is a one off script

Reply:
I think just hard code because it's a one-off thing